website/content/docs/editorial-workflows.md

```diff
@@ -0,0 +1,48 @@
+---
+group: Workflow
+weight: 20
+title: Editorial Workflows
+---
+
+
+By default, saving a post in the CMS interface pushes a commit directly to the publication branch specified in `backend`. However, you also have the option to edit the [Publish Mode](../configuration-options/#publish-mode), which adds an interface for drafting, reviewing, and approving posts. To do this, add the following line to your Decap CMS `config.yml`:
+
+```yaml
+# This line should *not* be indented
+publish_mode: editorial_workflow
+```
+
+From a technical perspective, the workflow translates editor UI actions into common Git commands:
+
+| Actions in Netlify UI     | Perform these Git actions                                                                                         |
+| ------------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| Save draft                | Commits to a new branch (named according to the pattern `cms/collectionName/entrySlug`), and opens a pull request |
+| Edit draft                | Pushes another commit to the draft branch/pull request                                                            |
+| Approve and publish draft | Merges pull request and deletes branch                                                                            |
+
+**Note:** Editorial workflow works with GitHub repositories, and support for GitLab and Bitbucket is [in beta](/docs/beta-features/#gitlab-and-bitbucket-editorial-workflow-support).
```

Comment:
This link will not work anymore. We could link to open issues with the EF tag. This way we might get some contributions. 

link: https://github.com/decaporg/decap-cms/issues?q=is%3Aissue+is%3Aopen+label%3A%22area%3A+ui%2Feditorial-workflow%22